baybe/kernels/kernels.py

```diff
@@ -0,0 +1,18 @@
+"""Collection of usable kernels."""
```

Comment:
"usable" is a bit unnecessary, no?

Reply:
Just wait until I start implementing more kernels, then you'll see that this is not unnecessary 😎 

Jokes aside, will change :D

Reply:
Changed